### PR TITLE
feat: home page and authentication added

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change logs
- Added home page
- Added Authentication

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import { Routes, Route, Outlet } from "react-router-dom";
 import "./App.css";
-import { Home } from "./pages";
+import { Home, Authentication, QuizCategories } from "./pages";
 import { Header, Footer } from "./components";
 function App() {
```
Comment (ravikumar0403) [inline src/App.jsx:5]: you can change this to es6

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -4,34 +4,32 @@ import { useTheme } from "../../context";
...
-							</span>
-						</li>
-					</ul>
-				</div>
+		<header className="header header-shadow flex-row justify-content-space-between align-center p-7">
+			<div className="brand-info flex-row justify-content-center align-center flex-gap-1 m-5">
+				<section>
+					<i className="fas fa-bars header-nav-icon"></i>
+				</section>
+				<Link to={"/"} className="no-link header-brand">
```
Comment (ravikumar0403) [inline src/components/Header/Header.jsx:12]: here `to` can be a normal string, need not enclose it in `{ }`

**Review by ravikumar0403 — APPROVED** [review]
really liked the way you have extracted out the logic with helpers functions in separate files. Good work

**Review by shashikantmaurya0007 — APPROVED** [review]
really liked the way you have extracted out the logic with helpers functions in separate files. Good work

**Pull request merged**